.sass-lint.yml

```diff
@@ -237,7 +240,7 @@ rules:
 
   # enforce how deeply a selector can be nested (defaults to 2)
   # https://github.com/sasstools/sass-lint/blob/master/docs/rules/nesting-depth.md
-  nesting-depth: 2
+  nesting-depth: 3
```

Comment:
does this change work? I think the syntax is 
```
nesting-depth:
    - 2
    -
      max-depth: 3
```
because the first line represents whether its reported as an error [2] or warning [1] and the second line is where you change the depth.